tool/gravity/cli/environ.go

```diff
@@ -207,6 +207,7 @@ func (environInitializer) newOperationPlan(
 	localEnv, updateEnv *localenv.LocalEnvironment,
 	clusterEnv *localenv.ClusterEnvironment,
 	leader *storage.Server,
+	userConfig interface{},
```

Comment:
nit: Same here, `userConfig` is unused.

Reply:
I'm matching the function signature to the interface for newOperationPlan